docs/rpc/api/core-node/get-burn-ops.example.json

```
@@ -0,0 +1,14 @@
+{
+  "PegIn": [
+    {
+      "amount": 1337,
+      "block_height": 217,
+      "burn_header_hash": "66b2f18530ccf0e57e82dd054cca9fa41bf5f645f7757171ad9c1546b86387e1",
+      "peg_wallet_address": "tb1pqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqkgkkf5",
+      "recipient": "S0000000000000000000002AA028H",
+      "recipient_contract_name": null,
```

FYI, I changed this format today after a discussion in #3486. The `recipient` is now represented as `PrincipalData`, and `recipient_contract_name` has been removed completely.

```suggestion
      "recipient": "S0000000000000000000002AA028H",
```

---

Yep, updated in 92ec39e4d36420fb9b5c966ac549a84ffc3417cc